src/Card/Card.js

```diff
@@ -1,15 +1,42 @@
 import './Card.css';
 import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Card({ name, casting_time, range, duration, desc, handleKnown, handleDelete }) {
+
+  const selectedSpell = {
+    key: name,
+    id: name,
+    name: name,
+    casting_time: casting_time,
+    range: range,
+    duration: duration,
+    desc: desc,
+    isKnown: false
+  }
 
-export default function Card({ name, casting_time, range, duration, desc }) {
   return (
     <div className='spell-card'>
       <h2>{name}</h2>
       <p>{casting_time}</p>
       <p>{range}</p>
       <p>{duration}</p>
       <p>{desc}</p>
-      <button>Learn</button>
+      <div className='button-box'>
+        <button className="learn-toggle" id='learn' onClick={() => handleKnown(selectedSpell)}>Learn</button>
```

Comment:
do you have both of these buttons rendering at once? that can be a good alternative to a ternary as well

Reply:
I do currently, I'm going to add button disabling based on the isKnown property so users can't click them improperly.